src/github.com/getlantern/lantern-ui/app/js/services.js

```diff
@@ -40,12 +88,33 @@ angular.module('app.services', [])
             $rootScope.originalList = entries;
           })
         }
-      }
+      },
+      'Stats': function(data) {
+        if (data.type != "peer") {
+          // Ignore anything that's not peers
```

Comment:
Good point, fixed.  Didn't matter much right now because all messages are peers, but good to fix.